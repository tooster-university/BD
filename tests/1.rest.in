{"open": {"database": "student", "login": "app", "password": "qwerty"}}
{"projects": {"timestamp": 67824186, "member": 4, "password": "GVBLTCFAZW"}}
{"projects": {"timestamp": 105574485, "member": 7, "password": "J5PXP"}}
{"actions": {"timestamp": 120493825, "member": 4, "password": "GVBLTCFAZW"}}
{"votes": {"timestamp": 145445615, "member": 7, "password": "J5PXP"}}
{"support": {"timestamp": 169735127, "member": 7, "password": "J5PXP", "action": 114, "project": 280, "authority": 413}}
{"projects": {"timestamp": 174341649, "member": 7, "password": "J5PXP"}}
{"upvote": {"timestamp": 228150748, "member": 4, "password": "GVBLTCFAZW", "action": 118}}
