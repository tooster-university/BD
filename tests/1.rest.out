{"status": "OK"}
{"status": "ERROR"}
{"status": "ERROR"}
{"status": "ERROR"}
{"status": "ERROR"}
{"status": "ERROR"}
{"status": "ERROR"}
{"status": "ERROR"}
